x-pack/packages/kbn-elastic-assistant-common/impl/llm/types.ts

```diff
@@ -38,3 +39,12 @@ export interface InvokeAIActionParamsSchema {
   functions?: ChatCompletionCreateParamsNonStreaming['functions'];
   signal?: AbortSignal;
 }
+
+export interface TraceOptions {
+  evaluationId?: string;
+  exampleId?: string;
+  projectName?: string;
+  runName?: string;
+  tags?: string[];
+  tracers?: LangChainTracer[];
```

Comment:
As noted when we paired, we should update this type to support our internal `APMTracer` type as well so we can just add it to the array and pass along `TraceOptions` wherever it needs to go. No change needed here, but something I made a note of for when I'm back working some eval changes. 👍